cmd/serverless-init/log/log.go

```diff
@@ -92,11 +96,48 @@ func SetupLog(conf *Config, tags map[string]string) {
 }
 
 func (cw *CustomWriter) Write(p []byte) (n int, err error) {
```

Comment:
make `Write` a facade so we can pass a custom `maxBufferSize` in for unit testing. previously, stdout would clutter the testing output due to large maxBufferSize